src/App.js

```diff
@@ -1,10 +1,56 @@
 import React from "react";
 import "./App.css";
+import { useState } from "react";
 
 function App () {
+const [score,setScore]=useState(0)
+const [increment,setIncrement]=useState(1)
+const [win,setWin]=useState("")
+const [display,setDisplay]=useState("")
+if (score>100){
+  setScore(score)
+  
+  
+}
+if (score<0){
+  setScore(0)
+}
```

Comment:
These lines of code cause it to constantly re-render once a player reaches a score over 100. Comment these lines out and you'll have a fully working game!